Makefile

```diff
@@ -245,21 +245,15 @@ darwin: ## Build for OSX
 	chmod +x build/darwin/$(NAME)
 
 .PHONY: release
-release: clean build test-slow-integration linux darwin win arm ## Release the binary
```

Comment:
I would probably delete the platform specific targets now - the `Makefile` is pretty busy already and any simplification would be great